README.md

```diff
@@ -27,20 +27,26 @@ React client to consume it
 }
 ```
 ## Getting Started
+1. Make sure you have `docker` and `docker-compose` installed on your machine
+    - To install docker: https://docs.docker.com/docker-for-mac/install/
+    - `brew install docker-compose`
 1. `docker-compose up --build -d`
 1. `docker ps` to find the container id of **conjugaciones-backend**
-1. `docker exec -it <conjugaciones-backend id>`
+1. `docker exec -it <conjugaciones-backend id> /bin/bash`
 1. from inside docker container:
     - `python` (to open python interpreter)
         - `from src.models import *`
         - `from app import db`
         - `db.create_all()`
-        - `db.sesion.commit()`
+        - `db.session.commit()`
         - `db.session.close()`
-        - `exit`
+        - `exit()`
     - `export PYTHONPATH=.`
     - `python scripts/verb_csv_importer.py ./verbs.csv`
 1. backend is now served on `localhost:8000`
+1. Navigate to the client folder in a separate window
+1. `yarn install`
+1. `yarn run build`
```

Comment:
Docker should do this for you in the `docker-compose.yml` file.
And build is the actual bundle-builder for deploying, not for hot-reloaded dev stuff

Reply:
idk I think something is missing somewhere because I agree that this should already be happening, but until I did this it didn't work locally. the only other thing I can think is that I didn't wait long enough for it to load up